src/api/SpecimenSearch.js

```diff
@@ -0,0 +1,23 @@
+import $ from 'jquery';
+
+function SpecimenSearch(query, callback) {
+    if (query) {
+        const endPoint = "api/v1/specimen/search";
+        const params = {
+            query: query,
+            pageSize: 25
+        }
+
+        $.ajax({
```

Comment:
I picked Ajax because I am familiar with using Jquery in PHP projects. In those cases I did not use a Javascript framework like React, but I now realize React is a persistent better approach for single page applications who regularly use fetches. Ajax is now replaced by the slightly different Axios call, also removed Ajax package.